jerry-core/ecma/base/ecma-helpers-number.c

```diff
@@ -37,16 +40,12 @@ ecma_number_pack (bool sign, /**< sign */
                   uint32_t biased_exp, /**< biased exponent */
                   uint64_t fraction) /**< fraction */
 {
-  const uint32_t fraction_pos = 0;
-  const uint32_t biased_exp_pos = fraction_pos + ECMA_NUMBER_FRACTION_WIDTH;
-  const uint32_t sign_pos = biased_exp_pos + ECMA_NUMBER_BIASED_EXP_WIDTH;
-
   JERRY_ASSERT ((biased_exp & ~((1u << ECMA_NUMBER_BIASED_EXP_WIDTH) - 1)) == 0);
   JERRY_ASSERT ((fraction & ~((1ull << ECMA_NUMBER_FRACTION_WIDTH) - 1)) == 0);
 
-  uint32_t packed_value = (((sign ? 1u : 0u) << sign_pos) |
-                           (biased_exp << biased_exp_pos) |
-                           (((uint32_t) fraction) << fraction_pos));
+  uint32_t packed_value = (((sign ? 1u : 0u) << ECMA_NUMBER_SIGN_POS) |
```

Comment:
The mentoined function's behaviour is similar, but not same with this, therefore (unfortunately) we can't use this in this case. Likewise at the other occurences.